### PR TITLE
Edit|Fast Data Single View Trigger Generator

### PR DESCRIPTION
**Pull request by danibix95**
## Pull Request Type

- [ ] Bugfix
- [X] Documentation content changes

## Description

This MR describe the changes made to the environment variables and custom strategy functions employed by Single View Trigger Generator service. In addition, a set of error codes with their description is introduced to easy debug operations.

## PR Checklist

- [X] The commit message follows our guidelines included in the [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md#how-to-submit-a-pr)
- [X] All tests of Lint, cspell and check-content pass. ([How to launch them?](../blob/main/CONTRIBUTING.md#content-checks))
- [X] No sensible content has been committed

### CODE REVIEW

File: docs/fast_data/configuration/single_view_trigger_generator.md [hunk 1]
```diff
@@ -88,3 +87,22 @@ The Kafka Projection Updates config map contains the `kafkaProjectionUpdates.jso
 
 Remember to copy/paste the mount path into the `KAFKA_PROJECTION_UPDATES_FOLDER` environment variable so the service can read the file.
 If you need more info on how to configure the `kafkaProjectionUpdates.json` file, please refer to the [Kafka Projection Updates](/fast_data/configuration/config_maps/kafka_projection_updates.md) page.
+
+## Error Codes
```
Comment (TheRusCompass) [inline docs/fast_data/configuration/single_view_trigger_generator.md:91]: May we add a reference to this error codes section also in the troubleshooting page?

Comment (danibix95) [reply docs/fast_data/configuration/single_view_trigger_generator.md:91]: Shall I move the error codes directly within a custom page, where all of them can be found (e.g. Fast Data Error Codes)?

Comment (danibix95) [reply docs/fast_data/configuration/single_view_trigger_generator.md]: I created a new page

Comment (TheRusCompass) [reply docs/fast_data/configuration/single_view_trigger_generator.md]: great! Let's add in the FAQ page the link to this error codes section. We can create a paragraph that says "if you want to better understand error codes go to this new page" or something like that. 

wdyt?

Comment (danibix95) [reply docs/fast_data/configuration/single_view_trigger_generator.md]: Done. I also reviewed the error codes to match our troubleshooting guidelines

**Review by TheRusCompass — APPROVED** [review]

**Pull request merged**